### PR TITLE
Fix & update man pages

### PR DESCRIPTION
**Pull request by eepp**

### CODE REVIEW

File: doc/man/lttng-load.1.txt [hunk 1]
```diff
@@ -76,16 +76,22 @@ option:-i, option:--input-path='PATH'::
     or a file, instead of loading them from the default search
     directories.
 
-option:--override-url::
-    Provides the equivalent of the option:--set-url option
-    of the man:lttng-create(1) command. The validity of the url override is
-    dependent of the type of session(s) to be loaded. Refer to
-    man:lttng-create(1) for the accepted URL format. This option applies
```
Comment (jgalar) [inline doc/man/lttng-load.1.txt]: The mention of the "accepted" URL format seemed important to me. We accept a (weird) format that specifies two ports (data/ctrl).

Maybe it's just me and referring to the `--set-url` is sufficient.

What do you think?

File: doc/man/lttng-add-context.1.txt [hunk 2]
```diff
@@ -46,13 +46,17 @@ can only be used in the Linux kernel tracing domain, while per-thread
...
-descriptor which is the same format as used by perf-record(1): a
-concatenation of the `Umask value` and `Event number` provided by the
-processor's manufacturer. The possible values for this field are
-processor-specific. The `<name>` field is used to associate a clearer
-name to the counter.
+`perf:cpu:raw:rN:NAME` (Linux kernel tracing domain) or
+`perf:thread:raw:rN:NAME` (user space tracing domain), with:
+
+`N`::
+    A hexadecimal event descriptor which is the same format as used
```
Comment (jgalar) [inline doc/man/lttng-add-context.1.txt]: A -> An

Comment (eepp) [inline doc/man/lttng-add-context.1.txt]: Well, I did not disagree with your suggestion to use _An_. It appears that, with some accents, when the first syllable of the word starting with _h_ is not stressed (which is the case with _hexadecimal_, the stress accent being on _dec_), it is correct to use _an_ before the word.

According to [this](http://english.stackexchange.com/questions/629/when-should-i-use-a-versus-an-in-front-of-a-word-beginning-with-the-letter-h), Queen Elizabeth II would say _an hexadecimal event descriptor_, while President Obama would say _a hexadecimal event descriptor_, although I doubt that one or the other would really say that, ever. Specifically, President Obama is known to be only interested in static PMU counters in his sessions of trace investigation.

**Comment by jgalar** [comment]
Merged, thanks!

**Pull request closed without merging**